security_groups.tf

```diff
@@ -1,28 +1,28 @@
 resource "aws_security_group" "redis_security_group" {
-  name        = "${format("%.255s", "tf-sg-ec-${var.name}-${var.env}-${data.aws_vpc.vpc.tags["Name"]}")}"
-  description = "Terraform-managed ElastiCache security group for ${var.name}-${var.env}-${data.aws_vpc.vpc.tags["Name"]}"
-  vpc_id      = "${data.aws_vpc.vpc.id}"
+  name        = format("%.255s", "tf-sg-ec-${var.name}-${var.env}-${local.vpc_name}")
+  description = "Terraform-managed ElastiCache security group for ${var.name}-${var.env}-${local.vpc_name}"
+  vpc_id      = data.aws_vpc.vpc.id
 
-  tags {
-    Name = "tf-sg-ec-${var.name}-${var.env}-${data.aws_vpc.vpc.tags["Name"]}"
+  tags = {
+    Name = "tf-sg-ec-${var.name}-${var.env}-${local.vpc_name}"
   }
 }
 
 resource "aws_security_group_rule" "redis_ingress" {
-  count                    = "${length(var.allowed_security_groups)}"
+  count                    = length(var.allowed_security_groups)
   type                     = "ingress"
-  from_port                = "${var.redis_port}"
-  to_port                  = "${var.redis_port}"
+  from_port                = var.redis_port
```

Comment:
Is this because we upgraded terraform that we dont need strings?

Reply:
haha yes exactly
https://github.com/DogParkLabs/terraform-aws-redis/pull/8